internal/launchr/tools.go

```diff
@@ -43,3 +43,13 @@ func GetTypePkgPathName(v interface{}) (string, string) {
 	}
 	return t.PkgPath(), t.Name()
 }
+
+// IsSELinuxEnabled checks if selinux is enabled on the system.
+func IsSELinuxEnabled() bool {
+	// @todo it won't actually work with a remote environment.
+	data, err := os.ReadFile("/sys/fs/selinux/enforce")
+	if err != nil {
+		return false
+	}
+	return string(data) == "1"
+}
```

Comment:
Looks like in some cases SELinux can be installed but this file not exist : 
```
➜  ~ sestatus    
SELinux status:                 disabled

➜  ~ ll /sys/fs/selinux/enforce                   
ls: cannot access '/sys/fs/selinux/enforce': No such file or directory

➜  ~ ll /sys/fs/selinux/                              
total 0
dr-xr-xr-x 2 root root 0 Sep 30 10:04 .
drwxr-xr-x 9 root root 0 Sep 30 10:04 ..
```

Reply:
Exactly
sestatus gives you disabled

Enable sestatus and you will have the file `/sys/fs/selinux/enforce`
You must also enable selinux in Docker driver. If not, labels won't be used at all

Reply:
```
$ cat /etc/docker/daemon.json
{
  "selinux-enabled": true
}
```